routers/likes.py

```
@@ -0,0 +1,49 @@
+from fastapi import APIRouter, Query
+from schemas.likes import LikeCreate
+
+router = APIRouter(prefix="/likes", tags=["likes"])
+
+@router.get("")
+async def get_my_likes(
+        page: int = Query(1, ge=1),
```

Ruff (0.14.11)

Unused function argument: `page`

(ARG001)